### ISSUE TITLE
No module named _sqlite3

### ISSUE THREAD

**Issue by Kickups** [pathmann/pyTSon#120]
Hey, I'm trying to import sqlite3 but it wont let me, I'm getting the response:

`  File "/home/UbuntuVM/.ts3client/plugins/pyTSon/lib/python3.5/sqlite3/dbapi2.py", line 27, in <module>
    from _sqlite3 import *
ImportError: No module named '_sqlite3'`

Any Idea what could be wrong?

**Comment by Kickups** [comment]
Also trying to install modules it's saying I'm using pip version 10, can I update this somehow or is it not necessary?

**Comment by pathmann** [comment]
Which version do you use and where did you get it from?
Might be a duplicate of #112

**Comment by Kickups** [comment]
Installed the VM just now and Installed the most recent version, not from the client, from the github page

**Comment by Kickups** [comment]
I looked at that Issue you linked above but I'm running Linux so I have no idea what files to move where. Only reason I decided to set this up on Linux is because I wanna learn a new operative system

**Comment by pathmann** [comment]
I didn't notice in your first post, that you are running Linux. I will look into the Linux build environment, if the sqlite module is built soon.

**Comment by pathmann** [comment]
Indeed, the sqlite3 module is not built on the linux build machines, I'll fix that for the next build (don't know, when, I need to adapt pythonqt to the new qt version for TeamSpeak Client >=3.3.0).

**Comment by Kickups** [comment]
Sounds good, thanks for taking your time! :)
